webrender/Cargo.toml

```diff
@@ -73,5 +73,5 @@ dwrote = "0.4.1"
 
 [target.'cfg(target_os = "macos")'.dependencies]
 core-foundation = "0.6"
-core-graphics = "0.16"
-core-text = { version = "11", default-features = false }
+core-graphics = "^0.17.1"
```

Comment:
why having ^ here?

Reply:
I wasn't sure what the default was when all subversions are specified, but I see that it's `^` by default, so I'll remove it.